examples/webhook-authz/README.md

```diff
@@ -52,17 +55,17 @@ kubectl apply -f ./authz.yaml -n clusterregistry
         - --authorization-webhook-cache-authorized-ttl=5m
         - --authorization-webhook-cache-unauthorized-ttl=30s
         volumeMounts:
-          ... existing volumeMounts ...
-          - name: bitesize
-            mountPath: /etc/bitesize
-          - name: ca-certificates
-            mountPath: /etc/ssl/certs
+        ... existing volumeMounts ...
+        - name: bitesize
+          mountPath: /etc/bitesize
+        - name: ca-certificates
+          mountPath: /etc/ssl/certs
 ...
       volumes:
       ... existing volumes ...
       - name: bitesize
-        configMap:
-          name: bitesize
+        secret:
+          secretName: bitesize
       - name: ca-certificates
         hostPath:
           path: /etc/ssl/certs
```

Comment:
Do you know why this is needed?

Reply:
I ran into a problem where the authorizer and/or clusterregistry didn't work unless I added a volume that pointed to the host's `/etc/ssl/certs`. I just updated my local deployment to not have this volume mounted and things still worked fine, so I'm wondering if this is an issue depending on how you create the cert that is used to secure communication between the webhook and the cluster registry API server. Perhaps I was using a cert that relied on one of the system certs?

It doesn't seem strictly necessary to making this run, so I think I'll remove it.